src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
```

Comment:
Body don't have any paddings or borders, so there is no sense to use box-sizing here (it's not inheriting by default). Instead better to use `*` selector for this purpose